core/tests/dispatch_integration_tests.rs

```diff
@@ -257,33 +299,25 @@ fn remote_delivery_bigger_than_buffer_messages_lazy_udp() {
     let system = system_from_network_config(net_cfg.clone());
     let remote = system_from_network_config(net_cfg);
 
-    let (ponger_named, ponf) = remote.create_and_register(BigPongerAct::new_lazy);
-    let poaf = remote.register_by_alias(&ponger_named, "custom_name");
-    let _ = ponf.wait_expect(Duration::from_millis(1000), "Ponger failed to register!");
-    let mut ponger_named_path =
-        poaf.wait_expect(Duration::from_millis(1000), "Ponger failed to register!");
-    ponger_named_path.via_udp();
-    let (pinger_named, pinf) =
-        system.create_and_register(move || BigPingerAct::new_lazy(ponger_named_path, 1500 * 3 / 4));
+    let (ponger, mut ponger_path) = create_and_run_big_ponger(&system, BigPongerAct::new_lazy());
+    ponger_path.via_udp();
+    let (pinger, pinger_complete_future) =
+        create_and_run_big_pinger(&remote, BigPingerAct::new_lazy(ponger_path, 1500 * 3 / 4));
```

Comment:
What's with the funky math? Should this perhaps be calculated in a variable so you can give it a name that someone other than you (e.g. future you :P) can actually comprehend? :D